mne/io/bti/bti.py

```diff
@@ -1169,18 +1188,24 @@ def _get_bti_info(pdf_fname, config_fname, head_shape_fname, rotation_x,
         if chan_4d.startswith('A'):
             chan_info['kind'] = FIFF.FIFFV_MEG_CH
             chan_info['coil_type'] = FIFF.FIFFV_COIL_MAGNES_MAG
-            chan_info['coord_frame'] = FIFF.FIFFV_COORD_DEVICE
+            chan_info['coord_frame'] = (
+                FIFF.FIFFV_COORD_DEVICE if convert else
+                FIFF.FIFFV_MNE_COORD_4D_HEAD)
```

Comment:
@Eric89GXL I remember it was like this, also in Jülich. The HCP BTI data are native.